Index: debian/control
===================================================================
--- debian/control	(revision 366)
+++ debian/control	(working copy)
@@ -3,7 +3,7 @@
 Priority: extra
 Maintainer: Samuel Bovée <samuel@ulteo.com>
 Build-Depends: debhelper (>= 5), cdbs, quilt,
- python (>= 2.6), python-central, python-svn,
+ python (>= 2.5), python-central, python-svn,
  libxml2-dev, libx11-dev, libxfixes-dev, libssl-dev, libpam0g-dev, libpulse-dev,
  libcupsys2-dev | libcups2-dev, libltdl3-dev, libfuse-dev
 Standards-Version: 3.8.3
Index: debian/rules
===================================================================
--- debian/rules	(revision 366)
+++ debian/rules	(working copy)
@@ -1,7 +1,7 @@
 #!/usr/bin/make -f
 
 DEB_PYTHON_SYSTEM := pycentral
-DEB_PYTHON_COMPILE_VERSION = 2.6
+DEB_PYTHON_COMPILE_VERSION = 2.5
 DEB_PYTHON_MODULE_PACKAGES = xrdp-python
 
 include /usr/share/cdbs/1/class/autotools.mk
